### ISSUE TITLE
Build error when building library

### ISSUE THREAD

**Issue by JaiSh12345** [raysan5/raylib#1016]
Please, before submitting a new issue verify and check:

 - [x] I tested it on latest raylib version from master branch
 - [x] I checked there is no similar issue already reported
 - [x] My code has no errors or misuse of raylib

### Issue description

I was trying to build raylib with `mingw32-make PLATFORM=PLATFORM_DESKTOP`, but getting an error.

### Environment

Platform: Windows 10 x64 processor
OpenGL version: How to check?
GPU details: Which would be appropriate?

### Issue Screenshot

Just copy-pasting the error here, in case it needs to be copied.
```
C:\raylib\raylib\src>mingw32-make PLATFORM=PLATFORM_DESKTOP
gcc -c rglfw.c -O1 -Wall -std=c99 -D_DEFAULT_SOURCE -Wno-missing-braces -Werror=pointer-arith -fno-strict-aliasing -Werror=implicit-function-declaration  -I. -Iexternal/glfw/include -Iexternal/glfw/deps/mingw -DPLATFORM_DESKTOP -DGRAPHICS_API_OPENGL_33
In file included from rglfw.c:62:0:
external/glfw/src/win32_init.c: In function '_glfwUpdateKeyNamesWin32':
external/glfw/src/win32_init.c:493:42: error: 'MAPVK_VSC_TO_VK' undeclared (first use in this function)
             vk = MapVirtualKey(scancode, MAPVK_VSC_TO_VK);
                                          ^
external/glfw/src/win32_init.c:493:42: note: each undeclared identifier is reported only once for each function it appears in
In file included from C:/Program Files (x86)/CodeBlocks/MinGW/include/windows.h:48:0,
                 from external/glfw/src/win32_platform.h:68,
                 from external/glfw/src/internal.h:185,
                 from external/glfw/src/context.c:28,
                 from rglfw.c:54:
external/glfw/src/win32_init.c: In function '_glfwPlatformInit':
external/glfw/src/win32_platform.h:167:48: error: '_WIN32_WINNT_VISTA' undeclared (first use in this function)
     _glfwIsWindowsVersionOrGreaterWin32(HIBYTE(_WIN32_WINNT_VISTA),   \
                                                ^
external/glfw/src/win32_init.c:571:14: note: in expansion of macro 'IsWindowsVistaOrGreater'
     else if (IsWindowsVistaOrGreater())
              ^
external/glfw/src/win32_window.c: In function 'updateFramebufferTransparency':
external/glfw/src/win32_platform.h:167:48: error: '_WIN32_WINNT_VISTA' undeclared (first use in this function)
     _glfwIsWindowsVersionOrGreaterWin32(HIBYTE(_WIN32_WINNT_VISTA),   \
                                                ^
external/glfw/src/win32_window.c:382:10: note: in expansion of macro 'IsWindowsVistaOrGreater'
     if (!IsWindowsVistaOrGreater())
          ^
external/glfw/src/win32_window.c: In function 'acquireMonitor':
external/glfw/src/win32_platform.h:164:48: error: '_WIN32_WINNT_WINXP' undeclared (first use in this function)
     _glfwIsWindowsVersionOrGreaterWin32(HIBYTE(_WIN32_WINNT_WINXP),   \
                                                ^
external/glfw/src/win32_window.c:542:13: note: in expansion of macro 'IsWindowsXPOrGreater'
         if (IsWindowsXPOrGreater())
             ^
external/glfw/src/win32_window.c: In function 'releaseMonitor':
external/glfw/src/win32_platform.h:164:48: error: '_WIN32_WINNT_WINXP' undeclared (first use in this function)
     _glfwIsWindowsVersionOrGreaterWin32(HIBYTE(_WIN32_WINNT_WINXP),   \
                                                ^
external/glfw/src/win32_window.c:569:13: note: in expansion of macro 'IsWindowsXPOrGreater'
         if (IsWindowsXPOrGreater())
             ^
external/glfw/src/win32_window.c: In function 'createNativeWindow':
external/glfw/src/win32_platform.h:170:48: error: '_WIN32_WINNT_WIN7' undeclared (first use in this function)
     _glfwIsWindowsVersionOrGreaterWin32(HIBYTE(_WIN32_WINNT_WIN7),    \
                                                ^
external/glfw/src/win32_window.c:1263:9: note: in expansion of macro 'IsWindows7OrGreater'
     if (IsWindows7OrGreater())
         ^
external/glfw/src/win32_window.c: In function '_glfwPlatformFramebufferTransparent':
external/glfw/src/win32_platform.h:167:48: error: '_WIN32_WINNT_VISTA' undeclared (first use in this function)
     _glfwIsWindowsVersionOrGreaterWin32(HIBYTE(_WIN32_WINNT_VISTA),   \
                                                ^
external/glfw/src/win32_window.c:1821:10: note: in expansion of macro 'IsWindowsVistaOrGreater'
     if (!IsWindowsVistaOrGreater())
          ^
external/glfw/src/wgl_context.c: In function 'swapBuffersWGL':
external/glfw/src/win32_platform.h:167:48: error: '_WIN32_WINNT_VISTA' undeclared (first use in this function)
     _glfwIsWindowsVersionOrGreaterWin32(HIBYTE(_WIN32_WINNT_VISTA),   \
                                                ^
external/glfw/src/wgl_context.c:322:13: note: in expansion of macro 'IsWindowsVistaOrGreater'
         if (IsWindowsVistaOrGreater())
             ^
external/glfw/src/wgl_context.c: In function 'swapIntervalWGL':
external/glfw/src/win32_platform.h:167:48: error: '_WIN32_WINNT_VISTA' undeclared (first use in this function)
     _glfwIsWindowsVersionOrGreaterWin32(HIBYTE(_WIN32_WINNT_VISTA),   \
                                                ^
external/glfw/src/wgl_context.c:349:13: note: in expansion of macro 'IsWindowsVistaOrGreater'
         if (IsWindowsVistaOrGreater())
             ^
Makefile:504: recipe for target 'rglfw.o' failed
mingw32-make: *** [rglfw.o] Error 1
```

### Code Example

Just building the library is causing this error.

**Comment by JaiSh12345** [comment]
Looks like this library is dead. Oh well...

**Comment by chriscamacho** [comment]
hmmm someone didn't leap to provide you with free support so the library must be dead, actually there is a thriving community, pity you didn't ask there...

**Comment by JaiSh12345** [comment]
Please do tell where and how to ask. I am new here.

**Comment by chriscamacho** [comment]
Try the raylib discord, see discord icon on raylib.com - this looks like a setup issue not a bug with raylib...
you might profit by reading this first https://codingkilledthecat.wordpress.com/2012/06/26/how-to-ask-for-programming-help/
